src/Spec/SanitizeSpec.php

```diff
@@ -37,6 +46,13 @@ class SanitizeSpec extends Spec
      */
     public function __invoke($subject)
     {
+
+        if ($this->subjectFieldIsBlank($subject) && $this->blank_field) {
+            $field = $this->field;
+            $replace_with = $this->blank_field;
+            $subject->$field = $subject->$replace_with;
+        }
+
```

Comment:
I have a feeling we should not call `$this->subjectFieldIsBlank($subject)`  if the user has explicitly set this to blank or not blank. So we need a flag for that , so both the ValidateSpec as in #134 should work. Not sure if I am over thinking / dump on this.